backend/src/submission/submission.controller.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  Controller,
+  Get,
+  NotFoundException,
+  Param,
+  UseGuards,
+} from "@nestjs/common";
+
+import { SessionGuard } from "src/auth/session.guard";
+
+import { SubmissionService } from "./submission.service";
+
+import { GetSubmissionsRes, Submission } from "~shared/types/api";
+
+@Controller("submissions")
```

Comment:
Let's park it under this route for now instead.
```suggestion
@Controller("/room/:roomId(\\w+)/submissions")
```